server/upstream/upstream-tarballs.txt

```diff
@@ -355,6 +355,7 @@ ibus-unikey:google:ibus-unikey
 iceauth:httpls:http://xorg.freedesktop.org/releases/individual/app/
 ico:httpls:http://xorg.freedesktop.org/releases/individual/app/
 icon-naming-utils:httpls:http://tango.freedesktop.org/releases/
+iio-sensor-proxy:httpls:https://github.com/hadess/iio-sensor-proxy/releases
```

Comment:
I merged too fast: this won't work, and you probably need https://github.com/openSUSE/osc-plugin-collab/pull/24 for this.